src/main/java/com/redhat/jenkins/plugins/cachet/SourceTemplate.java

```diff
@@ -0,0 +1,60 @@
+package com.redhat.jenkins.plugins.cachet;
+
+import hudson.Extension;
+import hudson.model.AbstractDescribableImpl;
+import hudson.model.Descriptor;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.Nonnull;
+import java.io.Serializable;
+
+public class SourceTemplate extends AbstractDescribableImpl<SourceTemplate> implements Serializable {
+
+    private static final long serialVersionUID = 4112681620416495146L;
+
+    private String cachetUrl;
+    private String label;
+    private boolean ignoreSSL;
```

Comment:
It is a matter of good style but those can be made final and setters removed.

Reply:
OK:)